core/src/main/java/org/fao/geonet/resources/CMISConfiguration.java

```diff
@@ -720,4 +739,53 @@ private String getServiceUrl(String baseUrl, String serviceUrl, String defaultSe
             return serviceUrl;
         }
     }
+
+    /**
+     * On certain cmis system the secondary property name/id is a bunch of codes that get changed each time the application is deployed.
+     *  Example from Open Text CMIS -   otCat:43102->otCat:43102:43102_5
+     *  This can cause issues for system that are re-deployed and the new codes need to be entered.
+     *  This function help convert a human-readable entry like
+     *     Catalogue->GeoNetwork Catalogue ID
+     *  to
+     *     otCat:43102->otCat:43102:43102_5
+     *  The parsing is done by using the property display name.
+     *
+     *  Note that using the display name means that it will stop on the next restart if the display name for the property is changed.
```

Comment:
I'm not sure if there is documentation in the GeoNetwork manual for configuring CMIS, if so it would be nice to add this kind of information.

Reply:
I'm guessing that it could go on this page?

https://github.com/geonetwork/doc/blob/fada79d9cb875e80de1441e48b609d08e0285a97/source/install-guide/customizing-data-directory.rst

I think we should also add a section for jcloud (old pr https://github.com/geonetwork/doc/pull/133) and cmis for external storage section.  I will try to revisit this in the new year.